protoc-gen-gogo/generator/generator.go

```diff
@@ -840,9 +840,10 @@ func (g *Generator) SetPackageNames() {
 	// Register the support package names. They might collide with the
 	// name of a package we import.
 	g.Pkg = map[string]string{
-		"fmt":   RegisterUniquePackageName("fmt", nil),
-		"math":  RegisterUniquePackageName("math", nil),
-		"proto": RegisterUniquePackageName("proto", nil),
+		"fmt":          RegisterUniquePackageName("fmt", nil),
+		"math":         RegisterUniquePackageName("math", nil),
+		"proto":        RegisterUniquePackageName("proto", nil),
+		"golang_proto": RegisterUniquePackageName("golang_proto", nil),
```

Comment:
Not sure I understand, I searched around a bit and couldn't find anywhere else where "proto" was registered. Any hints?